core/src/main/java/io/squashql/query/Functions.java

```diff
@@ -126,6 +126,46 @@ public static Measure plus(String alias, Measure a, Measure b) {
     return new BinaryOperationMeasure(alias, PLUS, a, b);
   }
 
+  public static Measure count(String alias, String field) {
+    return new AggregatedMeasure(alias, field, AggregationFunction.COUNT);
+  }
+
+  public static Measure count(String alias, Field field) {
+    return new AggregatedMeasure(alias, field, AggregationFunction.COUNT, null);
+  }
+
+  public static Measure countIf(String alias, String field, CriteriaDto criteriaDto) {
+    return new AggregatedMeasure(alias, new TableField(field), AggregationFunction.COUNT, criteriaDto);
+  }
+
+  public static Measure countIf(String alias, Field field, CriteriaDto criteriaDto) {
+    return new AggregatedMeasure(alias, field, AggregationFunction.COUNT, criteriaDto);
+  }
+
+  public static Measure countDistinct(String alias, String field) {
+    return new AggregatedMeasure(alias, field, AggregationFunction.COUNT, true);
+  }
+
+  public static Measure countDistinct(String alias, Field field) {
+    return new AggregatedMeasure(alias, field, AggregationFunction.COUNT, true);
+  }
+
+  public static Measure countDistinctIf(String alias, String field, CriteriaDto criteriaDto) {
+    return new AggregatedMeasure(alias, new TableField(field), AggregationFunction.COUNT, true, criteriaDto);
+  }
+
+  public static Measure countDistinctIf(String alias, Field field, CriteriaDto criteriaDto) {
+    return new AggregatedMeasure(alias, field, AggregationFunction.COUNT, true, criteriaDto);
+  }
```

Comment:
This method is not used. Could you create a test for that ? Similar to `testCountDistinct`.